.github/workflows/build-push-multi-arch.yml

```diff
@@ -66,14 +66,16 @@ jobs:
 
       - name: Parse inputs
         run: |
-          ARCH="${{ inputs.architecture }}"
+          ARCH="${{ inputs.architectures }}"
           ARCH_FLAGS=()
           if [[ ${ARCH} == "amd64" ]]; then
             ARCH_FLAGS+=(--platform=linux/amd64)
-          elif [[ ${ARCH} == "amd64 & arm64" ]]; then
+          elif [[ ${ARCH} == "arm64" ]]; then
+            ARCH_FLAGS+=(--platform=linux/arm64)
+          elif [[ ${ARCH} == "amd64,arm64" ]]; then
             ARCH_FLAGS+=(--platform=linux/arm64,linux/amd64)
           else
```

Comment:
Actually thinking about it more, maybe it's better to just have an input `platforms` instead of `architecture`, and pass them directly.

e.g. `platforms: linux/amd64,linux/arm64` -> `--platform="${{ inputs.platforms }}"`

If we take comma separated values but still parse the full string literally it's a strange in-between. E.g. `arm64,amd64` would not be valid in the current structure. If we pass the full platform list, it permits platforms in any order and also enables us to include different platforms if needed at some future point. Also, the next job will fail automatically ifs there is an invalid architecture so it also simplifies the logic.

Reply:
I was thinking about that too but was going for minimal change. Let's do it as you propose though :+1:

Reply:
Even the minimal change is a change in the API, e.g. will require downstream users to adapt the inputs, so we may as well break it properly. (or, if it was too much out of scope, then not change it at all; but I think it fits in this PR)